client/src/extension.ts

```diff
@@ -200,6 +204,7 @@ async function startSession (context: vscode.ExtensionContext, launchOptions: La
 
   // Wait until we're sure flix exists
   const flixFilename = await ensureFlixExists({ globalStoragePath, workspaceFolders, shouldUpdateFlix: launchOptions.shouldUpdateFlix })
+  handlers.setFlixFileName(flixFilename) 
```

Comment:
I think this is dangerous, because now this depends on things being loaded in the right order.

I think this should rather be set on the context object or something like that.